site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -23,9 +23,9 @@ If you want to know when the demo application is available, please [register you
 
 ### Install
 
-1.  Clone [the `blockprotocol` repository](https://github.com/blockprotocol/blockprotocol)
-1.  Run `yarn create-block-app yourblockname /path/to/yourblockname`
-1.  Switch to your new folder: `cd /path/to/yourblockname`
+1.  Move to a folder where you want to create your block
+1.  Run `npx create-block-app [your-block-name]` or `yarn create block-app [your-block-name]`
+1.  Switch to your new folder: `cd /path/to/[your-block-name]`
```

Comment:
We have instructed the user to supply a folder name rather than a path, so I think it should be:
```suggestion
1.  Switch to your new folder: `cd [your-block-name]`
```

Reply:
👍 changed